### PR TITLE
[prometheus-elasticsearch-exporter] Support extra arguments to the pod's container

### PR DESCRIPTION
**Pull request by mosheavni**
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [X] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-elasticsearch-exporter/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: prometheus-elasticsearch-exporter
-version: 4.9.0
+version: 4.9.1
```
Comment (monotek) [inline charts/prometheus-elasticsearch-exporter/Chart.yaml:4]: new feature should raise minor version

**Review by monotek — CHANGES_REQUESTED** [review]
Please also sign your commits and rebase with master

**Review by monotek — APPROVED** [review]

File: charts/prometheus-elasticsearch-exporter/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: prometheus-elasticsearch-exporter
-version: 4.9.0
+version: 4.10.0
```
Comment (monotek) [inline charts/prometheus-elasticsearch-exporter/Chart.yaml:4]:
```suggestion
version: 4.11.0
```

**Review by monotek — CHANGES_REQUESTED** [review]

**Review by monotek — CHANGES_REQUESTED** [review]
Please rebase / resolve merge conflicts and sign your commits.

**Comment by mosheavni** [comment]
i kinda regret trying to contribute here. too much bureaucracy and headache........

**Review by monotek — APPROVED** [review]

**Pull request merged**